### PR TITLE
Refactor saved-articles and lists to use v3 ES index

### PR DESCRIPTION
**Pull request by phamann**
This is the first major PR to refactor myft-page to use the more stable v3 ES index. 

- Create new `get-content` lib
    - Use signed fetch
    - Only request subset of fields from content model
- Create `v3-to-card' model to interface the new data to the model required by `next-card`
- Refactor `readAll` lists controller to make a single request to ES rather than one per list.
- Refactor saved-articles to use `get-content`
- Refactor read list to use `get-content`
- Bump the `article-primary-tag` lib to v2.0.0

### CODE REVIEW

**Comment by phamann** [comment]
I hoping this should have a substantial improvement on the performance of lists pages.

**Comment by phamann** [comment]
cc @matthew-andrews

File: server/controllers/lists/readAll.js [hunk 1]
```diff
@@ -1,28 +1,39 @@
...
+		return res;
+	}, {});
+};
+
+const hydrateListContent = (rawLists) => {
+	const listPromises = pluck('uuid', rawLists).map(getList);
+	let lists;
+
+	return Promise.all(listPromises)
+			.then(tap(listData => lists = listData))
```
Comment (laurieboyes) [inline server/controllers/lists/readAll.js]: ahhhhh took to a while to work out what this does but it's very clever and much nicer than passing ever growing objects along promise chains. nice.

File: server/controllers/saved-articles.js [hunk 2]
```diff
@@ -48,9 +50,10 @@ module.exports = function(req, res, next) {
 				viewModel.content = [];
 				render(viewModel);
 			} else {
-				return getLegacyContent(content.items, res.locals.flags).then(content => {
-					viewModel.content = content;
-					viewModel.content.forEach(a => a.isSaved = true);
+				return getContent(content.items.map(i => i.uuid)).then(content => {
+					viewModel.content = content
+											.map(v3toCard)
+											.map(tap(c => c.isSaved = true));
```
Comment (laurieboyes) [inline server/controllers/saved-articles.js]: although I don't understand the usage of tap here?

**Comment by laurieboyes** [comment]
:+1:

**Pull request merged**